agir/activity/migrations/0015_activity_is_pushed.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.1.8 on 2021-05-11 08:49
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("activity", "0014_auto_20210406_1712"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="activity",
+            name="is_pushed",
```

Comment:
sur le nom du champ je serai partisane d'enlever le is

C'est utile quand on doit indiquer qu'une variable est un boléen alors que c'est pas évident. Quand ça l'est c'est pas nécessaire, d'autant que ça n'est pas une convention du projet.

Le "is" ajoute la confusion parce que le présent pourrait faire croire qu'il s'agit d'un paramètre (la notification doit elle être poussée ou non). Si vraiment on devait mettre un verbe on mettrait "has_been_pushed".

Donc pour simplifier : "pushed".

(Désolée d'habitude je fais pas chier sur les noms de variable, sauf pour les champs de modèle car ils sont difficile à changer.)

Reply:
Pas de soucis, j'ai pris en compte le retour du coup !